docs/developer-docs/integrations/multi-chain/signatures-faq.md

```diff
@@ -0,0 +1,51 @@
+# Signatures FAQ
+
+## Does ICP support Threshold ECDSA signatures?
```

Comment:
Should we only say "ECDSA signatures"? I think Threshold ECDSA is specific to ICP

Reply:
agreed